### ISSUE TITLE
post-processing force/moments at the wall

### ISSUE THREAD

**Issue by tchatte3** [Exawind/nalu-wind#578]
Hi NALU-wind team, 

When I set up the post-processing section to compute  wall forces and moments in th NALU-wind yaml script, the program runs fine (without throwing any error), but I do not see the .dat files being generated.  Any help regarding this? I am attaching the .yaml file and .log file for reference. 

[abl_simulation.log](https://github.com/Exawind/nalu-wind/files/4587865/abl_simulation.log)
[abl_simulation.yaml.txt](https://github.com/Exawind/nalu-wind/files/4587867/abl_simulation.yaml.txt)

**Comment by sayerhs** [comment]
@tchatte3 Did you manually create the `results/` directory? There is a current limitation in nalu where it will silently fail if the directories are not created for post-processing. That is most likely the cause of your problem. 

Unrelated, I see from your input file that you have activated wall function but are using the force/moment post-processing for the non-wall function BC. You'll want to look in the source code and make sure this is what you want. Finally, in the default `master` branch, `use_wall_function: yes` for ABL simulations when using k-omega SST most likely isn't doing what you want it to do. So I hope you've modified the code for ABL simulations using RANS models.

**Comment by tchatte3** [comment]
@sayerhs, thanks for pointing this out. I really appreciate your help. I did not think about this since other restart and output files creates the directories. Related to the wall function, actually my goal is to see both the force/moments with and without wall functions, but for debugging purposes , I switched one of them off. So yes, I will be turning both the physics on for post-processing. 

Yes, I am aware of the fact that wall function in k-omega-SST currently is not designed to handle ABL simulations in NALU-wind. The .yaml file I shared is not for our production purpose and is just for a toy run. I really appreciate your feedback.
